zipline/assets/roll_finder.py

```diff
@@ -89,22 +89,21 @@ def get_rolls(self, root_symbol, start, end, offset):
         tc = self.trading_calendar
         sessions = tc.sessions_in_range(tc.minute_to_session_label(start),
                                         tc.minute_to_session_label(end))
+        freq = sessions.freq
         if first == front:
             curr = first_contract << 1
         else:
             curr = first_contract << 2
-        sess = sessions[-1]
-        while sess > start and curr is not None:
-            session_loc = sessions.searchsorted(sess)
+        session = sessions[-1]
```

Comment:
Super nitpicky, but can we add a newline for legibility here, to separate out the "initialization" from the loop?